WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -360,6 +360,7 @@ struct OrderForm_Previews: PreviewProvider {
 private extension ProductSelectorView.Configuration {
     static func addProductToOrder() -> ProductSelectorView.Configuration {
         ProductSelectorView.Configuration(
+            showsFilters: true,
```

Comment:
Do we need this property? Is it always true, or we have cases where we don't want to show the filters on the product selector view?

Reply:
At the moment is still needed for Product single-selection, this can be entirely removed shortly when multi-selection has been running in production for a few weeks, just to have an easy rollback if necessary at any point.

Reply:
Tracked here: https://github.com/woocommerce/woocommerce-ios/issues/9433